app/models/concerns/searchable_by_api.rb

```diff
@@ -0,0 +1,26 @@
+module SearchableByApi
```

Comment:
any specific reasons why we're creating concerns here? isn't this method only called by `Restaurant`?